.github/workflows/ci.yml

```diff
@@ -24,6 +27,17 @@ jobs:
       with:
         go-version: 1.17
 
+      # Don't run this since PR does not have access to secrets. 
+      # Let the make docker-build generate the temp. key
+    - name: Setup signing key
+      if: ${{ github.event_name != 'pull_request' }}
+      run: |
+        echo "$PRIVATE_KEY_PEM" > enclave-config/privatekey.pem
+        sha256sum enclave-config/privatekey.pem
+      shell: bash
+      env:
+        PRIVATE_KEY_PEM: ${{ secrets.PRIVATE_KEY_PEM }}
```

Comment:
@poussa Just for confirmation does the private key created with exponent 3 (`openssl genras -3`)?

Reply:
yes.